setup.cfg

```diff
@@ -51,7 +51,7 @@ ceph =
     lz4>=0.9.0
     tooz>=1.38
 ceph_recommended_lib =
-    cradox>=1.0.9
+    cradox>=1.2.0
```

Comment:
Why not just go for latest 1.3.2 ? Is there any distro stuck at 1.2.0 in particular?